benchmark/child_process/spawn-echo.js

```diff
@@ -1,12 +1,12 @@
 'use strict';
 var common = require('../common.js');
 var bench = common.createBenchmark(main, {
-  thousands: [1]
+  len: [1000]
```

Comment:
Actually, I see that you already set `n` to `1` when you launch the benchmarks, so maybe this should just be `n` and that's that?